app/jobs/check_layer_job.rb

```diff
@@ -3,6 +3,7 @@ class CheckLayerJob < ApplicationJob
 
   ##
   # @param [GeoMonitor::Layer] layer
+  # TODO: migrate this to the geo_monitor gem itself?
```

Comment:
Is this still a to do? Thanks

Reply:
It is; I added it as https://github.com/geoblacklight/geo_monitor/issues/17 but left a reminder here for later. Something to do after GBLv4 compatibility.